### ISSUE TITLE
Loop Builder filter feedback (elementor road map)

### ISSUE THREAD

**Issue by Michaelgimii** [elementor/elementor#21141]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team. @avivu, @arielk.

First of all, thank you for the road map video and also for the public release of the road map. This was really a great decision by the Elementor team.

Since you decide to work on the filter for the loop builder in the near future, I wanted to give my suggestion about this feature...

If I'm not mistaken, you plan to put the filter as a new section in the Loop Grid widget. But if you have this decision, I would say that it is a very limited solution. And it causes many limitations in the future. It's really important that the new features you add are expandable in the future.

### Describe the solution you'd like

Add a new widget called **Loop Filter** or any other name. Add the filters you currently want to add in this widget. So you will be able to develop this widget in the future. Filters such as price range, checkbox, radio and visual (color and image) which will be extremely important and useful for WooCommerce.

The filter for the Loop Grid widget will be incredibly important and a game changer. So please, if possible, check the jet smart filters plugin before publishing any filter. The filters in this filter have very powerful customization and meet the needs of a professional website to a large extent.

I hope that the filter widget that will be released in the future will remove the need for Elementor users from third-party plugins. Of course, I understand that it will not be possible to solve this need with just one update, but it is the right way.

thanks

**Comment by perks98** [comment]
I completely agree. If they want us to use the loop grid for products on a professional eCommerce site then having multiple filter options where you want them, like in a sidebar or popup is important.

**Comment by Michaelgimii** [comment]
Exactly. If the filter is integrated into the Grid loop widget, it will be very limited. For example, as you mentioned, we can't display the filters as a pop-up on mobile. And there are many reasons for that...

**Comment by 79ho3ein** [comment]
+1
From what I saw in the YouTube video, it looks like it's just going to add a filter bar section to the Loop Grid widget. But I think this is not a perfect solution. Rather, as you said, it is better to add it as a widget so that it can be developed in the future.
Also, if it is in the form of a widget, it gives users the possibility to place the filter in the desired location. For example, display filters in popup for mobile users.
Please don't forget that in the future, filters will be much needed in Elementor. So if you publish it as a widget from the beginning, it will be easier to develop it in the future.

cheers🥂

**Comment by FireRedDev** [comment]
+1 on this, it needs to be an external widget, at best maybe its even possible to stack filters

**Comment by portrayaloflife** [comment]
+1

**Comment by jcvikl** [comment]
+1 & filter pages
